.docker/images/nginx/x_frame_options.template

```diff
@@ -1,2 +1,2 @@
 # GOVCMS-1331: Peripheral pages are not protected again framing attacks
-add_header X-Frame-Options "$X_FRAME_OPTIONS";
+add_header X-Frame-Options "${X_FRAME_OPTIONS}";
```

Comment:
Should we set a default here in case the env var is not set?

Reply:
I think it's covered already -- a default is already set in `docker-compose.yml`

Reply:
`docker-compose.yml` is not included when run on lagoon environments only on local/CI.